kola/tests/misc/verity.go

```diff
@@ -117,10 +117,16 @@ func VerityCorruption(c cluster.TestCluster) {
 	}
 
 	// read the file back. if we can read it successfully, verity did not do its job.
-	out, err = c.SSH(m, "cat /usr/lib/os-release")
+	out, stderr, err := m.SSH("cat /usr/lib/os-release")
 	if err == nil {
 		c.Fatalf("verity did not prevent reading a corrupted file!")
 	}
+	for _, line := range strings.Split(string(stderr), "\n") {
```

Comment:
Will this log empty strings from a trailing newline?

Reply:
Nope, because `SSH()` trims trailing newlines.